build/compile.mk

```diff
@@ -40,10 +40,21 @@ compile-linux: deps-only
 	@for b in $(BINS); do \
 		OUTPUT_FILE="$(BUILD_DIR)linux/$$b" ; \
```

Comment:
We have this legacy error in all the compile targets, it should be OUTPUT_FILE="$(BUILD_DIR)/linux/$$b" and the same for every os, since there is @mkdir -p $(BUILD_DIR)/linux just over creating the right structure but the binary gets-created in the folder binlinux/

Reply:
I fixed it locally but then forgot to push :D. Thanks for pointing out. Fixed it.